src/libraries/LStratosphere.sol

```diff
@@ -22,9 +23,11 @@ library LStratosphere {
     ) internal view returns (bool isStratosphereMember, uint8 tier) {
         IStratosphere _stratosphere = IStratosphere(s.stratosphereAddress);
         uint256 _tokenId = _stratosphere.tokenIdOf(_address);
+
         if (_tokenId > 0) {
             isStratosphereMember = true;
-            tier = 0;
+            IRewardsController _rewardsController = IRewardsController(s.rewardsControllerAddress);
```

Comment:
what are the mechanism to ensure that `s.rewardsControllerAddress` is set? if it's not set, this snippet will revert. Is it ok?

Reply:
yes